public/main.js

```diff
@@ -0,0 +1,32 @@
+const clickEvent = function (event) {
+    const id = event.target.getAttribute('data-question-id');
+    const helpful = JSON.parse(event.target.getAttribute('data-helpful'));
+
+    const options = {
+        method: "POST",
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            id: id,
+            helpful: helpful
+        })
+    };
+    fetch("/rating", options)
```

Comment:
Where does the `fetch` function come from?

Reply:
fetch is built-in in modern browsers (i can use fetch without using Jquery or any other library)

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

Reply:
Cool! I didn't even know about that!